### ISSUE TITLE
[A11Y][Phone number] Country selector is not reachable anymore with screenreader

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#783]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

any

### OS Version

Android and iOS

### App version

beta 1.3.1 (692)

### Describe the issue

Country selector is not reachable with screen readers or keyboard, it worked on previous version, at least 0.6.0

### Expected Behavior

country selector shall be usable with screen readers and keyboard
